soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -258,4 +258,172 @@ struct uart_reg_t {
 #define NPCX_UFRCTL_RNEMPTY_EN                6
 #define NPCX_UFRCTL_ERR_EN                    7
 
+/*
+ * Multi-Input Wake-Up Unit (MIWU) device registers
+ */
+struct miwu_reg {
+	/* 0x000: Edge Detection 1 */
+	volatile uint8_t WKEDG1[1];
```

Comment:
More curious than anything: why are these arrays of size 1 instead of just non-array values?

(I am not asking you to change; I just want to learn the reason)

Reply:
At the beginning, the NPCX_WKEDG macro function is "NPCX_WKEDG_OFFSET(inst, n)    (inst->WKEDG1 + ((n) * 2L) + ((n) < 5 ? 0 : 0x1E))" Since we have use the offset directly in previous PR, I left them in case we need to access LOW_PND_WU register. The other registers can be replaced with macro functions.

Reply:
If we aren't using this, then let's remove it

Reply:
Done.